t/25-bugs.t

```diff
@@ -25,6 +25,8 @@ use OpenQA::Test::Database;
 use Test::More;
 use Test::Mojo;
 use Test::Warnings;
+use Minion::Job;
+use OpenQA::Task::Bug::Limit;
```

Comment:
After following @kraih's suggestion you don't need to explicitly use these modules anymore.